engine/app/components/citizens_advice_components/navigation.html.haml

```diff
@@ -0,0 +1,6 @@
+.cads-navigation-full-width-wrapper
+  %nav#cads-navigation.cads-navigation.js-cads-greedy-nav
+    %ul.cads-navigation__list
+      - links.each do |link|
+        %li.cads-navigation__list-item
+          = link_to link[:title], link[:url], title: link[:title], class: "cads-navigation__link"
```

Comment:
@davidsauntson Can you remember if there was a reason we include the `title` attribute for the navigation when it's the same as the link text?

Reply:
No it pre-dates both of us (https://github.com/citizensadvice/design-system/commit/5e903a52154094d79ee5e030f45e026ada022e9c#diff-398089f5ad0ebdc8766c211d098101d5a68b52d635a13bd2c975021524377a99) so happy for it to go